scripts/python/add_users_all_org_members_github_team.py

```diff
@@ -99,7 +80,34 @@ def run():
         print_stack_trace(message)
 
 
-print("Start")
-run()
-print("Finished")
-sys.exit(0)
+def main():
+    if len(sys.argv) == 2:
+        # Get the GH Action token
+        oauth_token = sys.argv[1]
+    else:
+        raise ValueError("Missing a script input parameter")
+
+    # Setup a transport and gql_client to interact with the GH GraphQL API
+    try:
+        transport = AIOHTTPTransport(
+            url="https://api.github.com/graphql",
+            headers={"Authorization": "Bearer {}".format(oauth_token)},
+        )
+    except Exception:
+        print_stack_trace("Exception: Problem with the API URL or GH Token")
+
+    try:
+        gql_client = Client(transport=transport,
+                            fetch_schema_from_transport=False)
+    except Exception:
+        print_stack_trace("Exception: Problem with the Client.")
+
+    github_service = GithubService(oauth_token, "ministryofjustice")
```

Comment:
kept org name

Reply:
Is it worth adding the ability to pass the org name as an argument? It might be overkill for this PR.